AirPlan/AirPlan/Controller/LoginViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  LoginViewController.swift
+//  AirPlan
+//
+//  Created by 이중엽 on 2023/07/10.
+//
+
+import UIKit
+import FirebaseCore
+import FirebaseAuth
+import GoogleSignIn
+
+class LoginViewController: UIViewController {
+    let loginView = LoginView()
+    let actionCodeSettings = ActionCodeSettings()
+    
+    override func loadView() {
+        self.view = loginView
+        loginView.initialSetup()
+        loginView.makeUI()
+        loginView.refresh()
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        loginView.loginButton.addTarget(self, action: #selector(loginButtonClicked), for: .touchUpInside)
+        loginView.signUpButton.addTarget(self, action: #selector(signUpButtonClicked), for: .touchUpInside)
+        loginView.signInGoogleButton.addTarget(self, action: #selector(signInGoogleButtonClicked), for: .touchUpInside)
+        loginView.testLogOutButton.addTarget(self, action: #selector(testLogOutButtonClicked), for: .touchUpInside)
+    }
+    
+    //일반 이메일 - 로그인 버튼 클릭 시
```

Comment:
```suggestion
    // 일반 이메일 - 로그인 버튼 클릭 시
```
기본적으로 주석을 생성하면 슬래시 뒤에 공백 하나가 있습니다. 다른 주석들도 체크하여 준수해주세요!